components/l10n/common/region_code_feature.cc

```diff
@@ -0,0 +1,14 @@
+/* Copyright (c) 2023 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at https://mozilla.org/MPL/2.0/. */
+
+#include "brave/components/l10n/common/region_code_feature.h"
+
+namespace brave_l10n {
+
+BASE_FEATURE(kFetchResourcesByRegionCodeFeature,
```

Comment:
GeoRegionCode (but do see other comment about country code usage)

Reply:
renamed to `kFetchResourcesByCountryCodeFeature`